poc_iot_verifier/src/main.rs

```diff
@@ -59,6 +60,9 @@ impl Server {
         let pool = settings.database.connect(2).await?;
         sqlx::migrate!().run(&pool).await?;
 
+        // Create the density scaler query messaging channel
+        let (density_tx, density_rx) = density_scaler::query_channel(50);
```

Comment:
What is the advantage of having the verifier communicate with the density scaler over message channels ?  Why not just instantiate it from within the verifier runner for example ( like what I do with the DenyList )?  Is there a distinct benefit with the message channels approach ?

Reply:
mostly just my erlang/elixir "communicating processes" pattern that made sense to me for an independent process operating within the boundaries of the verifier but ultimately doing separate side work. also, conceptually it lets me develop it more independently of the verifier itself and could theoretically be independently tested more easily if i had a decent way to mock the gateway info stream from the node follower